dspace-api/src/main/java/org/dspace/eperson/EPerson.java

```diff
@@ -487,7 +487,12 @@ public static int searchResultCount(Context context, String query)
             throws SQLException
     {
         String s, t = "", theQuery = "";
-
+        String sDotText_value;
+        if(DatabaseManager.isOracle()) {
+            sDotText_value= "cast(substr(m.text_value,1,200) as varchar2(200))";
+        } else {
+            sDotText_value = "m.text_value";
```

Comment:
It looks like the `sDotText_value` is set, but never actually used anywhere? Is there code missing from this PR?